cmd/crictl/image.go

```diff
@@ -73,13 +78,9 @@ var pullImageCommand = cli.Command{
 			return err
 		}
 
-		var auth *pb.AuthConfig
-		if context.IsSet("creds") {
-			var err error
-			auth, err = getAuth(context.String("creds"))
-			if err != nil {
-				return err
-			}
+		auth, err := getAuth(context.String("creds"), context.String("auth"))
+		if err != nil {
+			return err
 		}
 		var sandbox *pb.PodSandboxConfig
 		if context.IsSet("pod-config") {
```

Comment:
declaration of "err" shadows declaration at cmd/crictl/image.go:81 (vetshadow)

Reply:
That is super annoying, we ignore that error in containerd. :P

Reply:
Done